project/Publish.scala

```diff
@@ -49,8 +49,8 @@ object NoPublish extends AutoPlugin {
 
   override def projectSettings = Seq(
     publishArtifact := false,
-    publish := (),
-    publishLocal := ()
+    publish := {},
+    publishLocal := {}
```

Comment:
() is refered to as the "unit literal" so technically := () IMO is nicer... :-) 
I don;t mind merging as is though, thanks :)